crates/ruff_python_formatter/src/comments/placement.rs

```diff
@@ -991,14 +991,23 @@ fn handle_dict_unpacking_comment<'a>(
     .skip_trivia();
 
     // we start from the preceding node but we skip its token
-    if let Some(first) = tokens.next() {
-        debug_assert!(matches!(
-            first,
-            Token {
-                kind: TokenKind::LBrace | TokenKind::Comma | TokenKind::Colon,
-                ..
-            }
-        ));
+    while let Some(first) = tokens.next() {
```

Comment:
Could we add `{.text_len()` on line 982 instead of skipping over the `{`  in the iteration?

Reply:
didn't work for me, but this didn't work